src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div 
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__ph">
+        <img 
+          class="card__ph photo" 
+          src="./images/imac.jpeg" 
+          alt="imac">
+      </div>
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="rating">Отзывов: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="text-price">Цена:</p>
+        <p class="sum-price">69 999 грн</p>
+      </div>
```

Comment:
I guess, you can create a separate block here
```suggestion
      <div class="card__price price">
        <p class="price__text-price">Цена:</p>
        <p class="price__sum-price">69 999 грн</p>
      </div>
```

Reply:
I have deleted default style.css, and create new css file for each block, but npm test didn't run properly with such code, thats why Alisa advised me to add all styles to style.css, especially for this task, and push new commit on GH.